source/reference/api/start.txt

```diff
@@ -111,6 +127,64 @@ Request Body Parameters
 
        Default value is ``false``.
 
+
+.. _c2c-api-start-sharding:
+
+Sharding Parameters
+~~~~~~~~~~~~~~~~~~~
+
+To synchronize from a replica set to a sharded cluster, set the 
+``sharding`` option to shard collections on the destination cluster.
+
+The ``sharding`` option has the following parameters:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 15 15 70
+
+   * - Parameter
+     - Type
+     - Description
+
+   * - ``createSupportedIndexes``
+     - boolean
+     - Optional. Sets whether synchronization creates a supporting index 
+       for the shard key, if none exists.  Defaults to ``false``.
+
+   * - ``shardingEntries``
+     - array of documents
+     - Required. Sets the namespace and key of collections to shard 
```

Comment:
The array is required but can be empty, is it something that needs to be mentioned or not ?

Reply:
Fixed